File: app/src/main/java/ru/vincetti/vimoney/data/models/Account.java
package ru.vincetti.vimoney.data.models;

public class Account {
    private String name;
    private String type;
    private int sum;

    public Account(String name, String type, int sum) {
        this.name = name;
        this.type = type;
        this.sum = sum;
    }

    public String getName() {
        return name;
    }

    public String getType() {
        return type;
    }

    public int getSum() {
        return sum;
    }
}

File: Core/src/main/java/ru/vincetti/modules/core/models/Account.kt
package ru.vincetti.modules.core.models

@Suppress("DataClassShouldBeImmutable")
data class Account(
    var id: Int = DEFAULT_CHECK_ID,
    var name: String? = "",
    var type: String = ACCOUNT_TYPE_CASH,
    var sum: Int = 0,
    var currency: Int = DEFAULT_CURRENCY,
    var extraKey: String = "",
    var extraValue: String = "",
    var color: String = DEFAULT_COLOR,
    var isArchive: Boolean = false,
    var needAllBalance: Boolean = true,
    var needOnMain: Boolean = true
) {

    companion object {
        const val ACCOUNT_TYPE_CASH = "cash"
        const val ACCOUNT_TYPE_DEBIT = "debit"
        const val ACCOUNT_TYPE_CREDIT = "credit"

        const val DEFAULT_CHECK_ID = 0
        const val DEFAULT_COLOR = "#164fc6"
        const val DEFAULT_CURRENCY = 810
    }
}
